packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -68,6 +68,15 @@ describe('plugin-meetings', () => {
       assert.isFalse(audio.isSelf());
     });
 
+    it('does not consider remote mute status - isLocallyMuted', async () => {
```

Comment:
minor thing, but to follow the convention used in the existing tests, it would be better to wrap this test with a `describe('#isLocallyMuted')`